src/shared/views/topMenu/TopMenuHeader/TopMenuHeader.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+import styles from './topMenuHeader.css'
+
+const TopMenuHeader = (props) => {
+  const titleTransform = (location) => {
+    const title = location.split('/')
+    if (title[0] === '') return 'Home'
+    return title[0].charAt(0).toUpperCase() + title[0].slice(1)
```

Comment:
When using `charAt()` try use `charCodeAt()` it is much faster. And usually works the same.

Doesn't really matter what much, but just something to keep in the back of your head.

>  I realized I had made one big mistake: I used string.charAt(index) to take the characters from the string and evaluated them one by one. Even if inline strings can be used, this will give some overhead and every time I compare that character against another, the JS engine has to assume I'm doing a comparison between two random strings. So I switched the implementation to use string.charCodeAt(index) instead, and I instantly achieved a >75% performance improvement (commit):

>http://www.arendjr.nl/2016/09/how-i-made-text-clipper-fastest-html.html